packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -108,6 +108,7 @@ available:
 * `phone`: Allow signing in using the phone number that is associated with the account.
 * `preferredUsername`: Allow signing in with an alternate user name that the user can change at any time. However, this
   is not available if the `username` option is not chosen.
+* `caseSensitive`: When false, allow signing in without checking the capitalization of the alias. Default is true.
```

Comment:
This doesn't belong to this list. This list is providing the 4 different identifiers that users can use to sign in; `caseSensitive` is not one of them.

Move this to a paragraph just below this list.